tutorials/run-symfony-on-appengine-flexible/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Run Symfony on Google App Engine Flexible Environment
+description: Learn how to deploy a Symfony app to Google App Engine flexible environment.
+author: bshaffer
+tags: App Engine, Symfony, PHP
+date_published: 2017-03-15
+---
+## Symfony
+
+> [Symfony][symfony] is a set of PHP Components, a Web Application framework, a
```

Comment:
Oh, I see. They weren't on their [homepage](https://symfony.com/). In any case, IMHO I don't think they need to be capitalized since they aren't special terms in this context.